packages/react-server/src/ReactFizzServer.js

```diff
@@ -483,6 +484,48 @@ function renderSuspenseBoundary(
   // context switching. We just need to temporarily switch which boundary and which segment
   // we're writing to. If something suspends, it'll spawn new suspended task with that context.
   task.blockedBoundary = newBoundary;
+  if (
+    enableCPUSuspenseFizz &&
+    typeof props.unstable_expectedLoadTime === 'number'
+  ) {
+    // Create task to Render the fallback now
+    task.blockedSegment = boundarySegment;
+    try {
+      renderNode(request, task, fallback);
+      boundarySegment.status = COMPLETED;
+
+      // Create task to render the content
+      const contentTask = createTask(
+        request,
+        content,
+        newBoundary,
+        contentRootSegment,
+        fallbackAbortSet,
+        task.legacyContext,
+        task.context,
+        task.treeContext,
+      );
+      if (__DEV__) {
+        contentTask.componentStack = task.componentStack;
+      }
+      // Render the content in the next tick, since it is large.
+      setImmediate(() => {
```

Comment:
This will roughly be what we need to do but we'll need to abstract it out a bit (e.g. using the scheduleWork host config since not environment will have setImmediate nor do exactly the right thing) and think more about the sequencing.

How will this play with other things that use this technique? Just deferring it doesn't really say anything about the priority relative to other things we might work on. For example tasks that get pinged in microtasks later in this macro-task.

I think we'll instead want to add this to a different array, and then decide when this array gets worked on, and schedule a single callback from the end of performWork if we still have more things to work on.

Kind of like how in Fiber we always schedule a new callback at the end given the priority of whatever we have remaining to work on.